src/main/java/prestudy/team4/board/dto/CustomUserDetails.java

```diff
@@ -0,0 +1,62 @@
+package prestudy.team4.board.dto;
+
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+import prestudy.team4.board.domain.UserEntity;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+public class CustomUserDetails implements UserDetails {
+
+    private UserEntity userEntity;
+
+    public CustomUserDetails(UserEntity userEntity) {
+        this.userEntity = userEntity;
+    }
+
+    @Override
+    public Collection<? extends GrantedAuthority> getAuthorities() {
+
+        Collection<GrantedAuthority> collection = new ArrayList<>();
+
+        collection.add(new GrantedAuthority() {
+            @Override
+            public String getAuthority() {
+                return userEntity.getRole();
+            }
+        });
+
+        return collection;
+    }
+
+    @Override
+    public String getPassword() {
+        return userEntity.getPassword();
+    }
+
+    @Override
+    public String getUsername() {
+        return userEntity.getUsername();
+    }
+
+    @Override
+    public boolean isAccountNonExpired() {
+        return false;  // 원래 설정대로 계정 만료 상태로 복원
```

Comment:
```suggestion
        return true; // true가 기본값이라고 해요.
```
`isAccountNotExpired` 메서드는 계정 만료를 확인하는 메서드에요.
여기서 false를 반환하면 해당 계정은 항상 만료된 상태로 간주해서 아무도 로그인을 할 수가 없게 되어요. false로 둔 특별한 이유가 있으실까요 ?!